modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7245,6 +7245,7 @@ field-reference-serves-as-a-frienldy-identifier=Field reference serves as a frie
 field-set=Field Set
 field-sets=Field Sets
 field-settings=Field Settings
+field-successfully-added=Field successfully added.
```

Comment:
@diogomelo09 please change to `The field was successfully added.`

Reply:
Done!! :heavy_check_mark: